src/main/java/core/types/HeartRate.java

```diff
@@ -0,0 +1,18 @@
+package core.types;
+
+/**
+ * Created by mikeecb on 17/10/2015.
+ */
+public class HeartRate {
```

Comment:
I made a seperate data type so that it is obvious what we're returning and to make it immutable. Why would an interface be better?